src/chains/ethereum/ethereum/src/helpers/gas-estimator.ts

```diff
@@ -1,12 +1,36 @@
-import { BN } from "ethereumjs-util";
+import BN from "bn.js";
 import { RuntimeError, RETURN_TYPES } from "@ganache/ethereum-utils";
 import { Quantity } from "@ganache/utils";
+import { RunTxOpts, RunTxResult, VM } from "@ethereumjs/vm";
+import { EVM } from "@ethereumjs/evm";
+import type { InterpreterStep } from "@ethereumjs/evm/";
+import { RuntimeBlock } from "@ganache/ethereum-block";
 
-const bn = (val = 0) => new (BN as any)(val);
+const bn = (val = 0): BN => new BN(val);
```

Comment:
```suggestion
const bn = (val = 0) => new BN(val);
```
The return type is implicit and reasonably obvious with the BN ctor right there

Reply:
`new BN(val)` returns type `any`, so that's why I'm setting explicitly to return a BN